emperor/support_files/js/abc-view-controller.js

```diff
@@ -183,5 +183,14 @@ define([
     throw Error('Not implemented');
   };
 
+  /**
```

Comment:
- When switching to the parallel plot view, it would be great if the camera could try to fit most of the plot or at least 5 dimensions of the plot on screen.
    - Simiarly, when clicking "recenter camera" it should go to that view that fits ~5 dimensions.